addons/account/wizard/account_move_reversal.py

```diff
@@ -96,6 +96,7 @@ def _compute_from_moves(self):
 
     def _prepare_default_reversal(self, move):
         reverse_date = self.date if self.date_mode == 'custom' else move.date
+        bank_ids = move.commercial_partner_id.bank_ids.filtered(lambda bank: bank.company_id is False or bank.company_id == move.company_id)
```

Comment:
Maybe this is slightly more readable and better for performance? (and use the `partner_bank_id.id` immediately below)
```suggestion
        partner_bank_id = self.env['res.partner.bank'].search([
            ('partner_id', '=', move.commercial_partner_id.id),
            ('company_id', 'in', [move.company_id.id, False])
        ], limit=1)

Reply:
Nicely spotted! :+1: